src/ui/public/vislib/lib/tilemap_settings.js

```diff
@@ -0,0 +1,135 @@
+import uiModules from 'ui/modules';
+import _ from 'lodash';
+import marked from 'marked';
+import url from 'url';
+
+marked.setOptions({
+  gfm: true, // Github-flavored markdown
+  sanitize: true // Sanitize HTML tags
+});
+
+
+uiModules.get('kibana')
+  .service('tilemapSettings', function ($http, mapsConfig, $sanitize) {
```

Comment:
@ppisljar agreed, I'm not a fan of this spot either. Perhaps we can move it out when we do #9441?

@spalger do you have any recommendation as to where to move it? I kept it in `.directives` for a while, since that seems somewhat of a catch-all.